### ISSUE TITLE
Model offset can be set outside the range (-2,2) only if its current value is within the range (-2,2)

### ISSUE THREAD

**Issue by tkuchida** [opensim-org/opensim-gui#734]
### Steps to reproduce
- open a model
- right-click the model in the *Navigator* and select  *Display > Model offset...*
- set *Offset X* to 3 by typing "3" in the first textbox and clicking *OK*
- open *Model offset* dialog again and replace "3" with "4"

### Expected result
Model moves to x=3 when "3" is entered in the *Offset X* box. Model moves to x=4 when "3" is replaced with "4".

### Actual result
Not sure what the underlying logic is, but it seems that the model moves beyond the range (-2,2) only if its current value is within the range (-2,2). This behavior is unexpected. To be consistent with the *Coordinates* panel, it seems that the user should be able to specify arbitrary offsets, and whether the offset is applied should not depend on the model's current offset.

### Environment and GUI version
AppVeyor artifact OpenSim-d8bfa962-2018-05-09-win64.exe

**Comment by aymanhab** [comment]
@tkuchida I agree there could be a bug in the unexpected behavior you described. Likely in not reverting the number to within bounds or alerting the user it is not in range.
There's a difference, however, between coordinates (that are part of the model and so have meaningful physical bounds) vs arbitrary offsets used for visualization convenience. In fact these could be arbitrary large (depending on units). The numbers 2, -2 were fixed based on reasonable number of models but can be changed to another more flexible scheme (e.g. 2* number of open models).  Please let me know what you think.

**Comment by tkuchida** [comment]
> There's a difference, however, between coordinates (that are part of the model and so have meaningful physical bounds) vs arbitrary offsets used for visualization convenience.

I was just trying to understand the rationale for the current design. I assumed that you want the offset dialog box to have the same aesthetics as the *Coordinates* panel. If so, a slider seems fine if you think that will be most convenient, but I think it should still be possible to enter any desired value in the textbox.

> The numbers 2, -2 were fixed based on reasonable number of models but can be changed to another more flexible scheme (e.g. 2* number of open models).

Aside from potential aesthetic considerations, I don't really see the benefit of providing users with sliders that have arbitrary bounds. Perhaps the user has 2 models open and wants to move model *A* out of the way while model *B* animates. In that case, you would need to know the volume through which model *B* will move so model *A* can be offset outside this range. My use case was that I wanted to pose several models in a row---and it is impossible to know how far apart I want to space the models.

**Comment by aseth1** [comment]
I would prefer not to have the sliders to move the offset. IMHO they are generally not that helpful. I like entering a value so I can position and remember the layout of multiple models. In arranging the layout it would be helpful to move the entire model using the same position tools to move markers, lights, etc... and see the offset values update accordingly.
